src/components/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { Person } from '../../types';
+import { PersonLinks } from '../PersonLinks/PersonLinks';
+
+type Props = {
+  people: Person[];
+};
+
+export const PeopleTable: React.FC<Props> = ({ people }) => {
+  return (
+    <table
+      data-cy="peopleTable"
+      className="table is-striped is-hoverable is-narrow is-fullwidth"
+    >
+      {people.length > 0 ? (
+        <>
+          <thead>
+            <tr>
+              <th>Name</th>
+              <th>Sex</th>
+              <th>Born</th>
+              <th>Died</th>
+              <th>Mother</th>
+              <th>Father</th>
+            </tr>
+          </thead>
+          <tbody>
+            {people.map((person: Person) => {
+              return (
+                <PersonLinks
+                  person={person}
+                  key={person.slug}
```

Comment:
Ensure that `person.slug` is a unique identifier for each `Person` object. The `key` prop should be unique to prevent issues with React's reconciliation process. If `slug` is not guaranteed to be unique, consider using another unique identifier.